package-lock.json

```diff
@@ -8284,20 +8284,20 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "",
+          "resolved": false,
```

Comment:
Can you reset this file to latest please

Reply:
Otherwise LGTM

Reply:
Done!